prow/jobs_config/knative/infra.yaml

```diff
@@ -0,0 +1,15 @@
+org: knative
+repo: infra
+branches: [main]
+image: gcr.io/knative-tests/test-infra/prow-tests:v20230331-37ec75c1
```

Comment:
image path is still pointing at test-infra

Reply:
https://github.com/knative/test-infra/pull/3784#discussion_r1158512034